schema.sql

```diff
@@ -0,0 +1,3 @@
+CREATE DATABASE vet_clinic
+create table animals (id integer, name varchar(20), data_of_birth date,escape_attempts integer,neutered boolean,weight_kg decimal);
```

Comment:
[OPTIONAL]
- I suggest you make the id column not null and autoincremented.
`id int NOT NULL IDENTITY(1,1)`
 And would also advise making it a primary key
`PRIMARY KEY (id)`